nestkernel/connection_manager.cpp

```diff
@@ -1087,8 +1087,8 @@ nest::ConnectionManager::get_connections( DictionaryDatum params ) const
 
 // Helper method, implemented as operator<<(), that removes ConnectionIDs from
 // input deque and appends them to output deque.
-static inline std::deque< nest::ConnectionID >& operator<<(
-  std::deque< nest::ConnectionID >& out,
+static inline std::deque< nest::ConnectionID >&
+join_deques( std::deque< nest::ConnectionID >& out,
```

Comment:
I don't have enough experience with the code base to review the patch, I think. I agree that the name should reflect what the code does to nest rather than what it does to the data structure.

The original code uses static, so it makes sense to keep it for this patch. If it's unneeded, then it's probably better to create a separate issue to change that.